apps/native-component-list/package.json

```diff
@@ -6,6 +6,9 @@
   "private": true,
   "main": "__generated__/AppEntry.js",
   "scripts": {
+    "start": "expo start",
```

Comment:
Remove these -- we want the way to run NCL to be the same as running other Expo projects.